client/stylesheets/_modal.scss

```diff
@@ -843,9 +843,68 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	margin-top: 75px !important;
 }
 
+.modFeedbackPageTwo {
+	margin-top: 50px !important;
+	padding: 10px 40px !important;
+}
+
+.modFeedbackPageTwo h2 {
```

Comment:
Merge rule `.modFeedbackPageTwo h2` with rule on line 846<br>Selector `modFeedbackPageTwo` should be written in lowercase with hyphens